.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--
+  Thanks for submitting a pull request!
+  We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request.
+
+  Please mark this PR as "Draft" if it is not ready for review.
+-->
+
```

Comment:
Also, can we standardize the PR naming/wording convention a little bit if that has ever been an issue?

Reply:
ok will add the convention reference to the template 👍